client/batch_test.go

```diff
@@ -1746,14 +1745,17 @@ func mutexNilClearID(t *testing.T, c *test.Cluster, client *Client) {
 	}
 	items := resp.Result().Row().Columns
 	// delete item 0
-	b.Add(
+	err = b.Add(
 		Row{
 			ID:     items[0],
 			Values: []interface{}{nil},
 			Clears: map[int]interface{}{0: nil},
 		},
 	)
-	b.Import()
+	assert.Nil(t, err)
```

Comment:
assert has a `NoError()` method that we should probably use instead of `Nil()`
The difference would be that it takes a typed error. I think it's better to use that explicit method.

Reply:
Updated to NoError